_data/tools.yml

```diff
@@ -1961,18 +1961,24 @@
 - name: Unity Catalog
   anchor: unity-catalog
   category: data-lake
-  owner: trinodb
+  owner: other
   logo: /assets/images/logos/unity-catalog.png
   logosmall: /assets/images/logos/unity-catalog-small.png
   description: |
+    Note that the support for Unity Catalog via the Thrift protocol is
+    deprecated by Unity Catalog and is removed from Trino 473 and later. Users
+    must retrieve and update the code from the source code repository, use an
+    old Trino version, or provide an implementation for a new Unity Catalog
+    integration.
+
     Unity Catalog is a unified governance solution for data and AI assets in the
     cloud. It provides a centralized metadata store, fine-grained access
     controls, and audit logging to help manage and secure your data. Unity
     Catalog simplifies data discovery, access, and sharing across your
     organization, ensuring compliance and enhancing collaboration.
 
     Use Unity Catalog with the Trino Delta Lake, Hive, or Iceberg connectors and
-    the configuration for a Thrift metastore.
+    the configuration for a Thrift metastore with Trino vers
```

Comment:
Fix uncompleted sentence. Note that Iceberg connector still can access Unity catalog using Iceberg REST Catalog protocol.

Reply:
Updated..